pkg/utils/rebaseTodo.go

```diff
@@ -32,3 +38,88 @@ func WriteRebaseTodoFile(fileName string, todos []todo.Todo) error {
 	}
 	return err
 }
+
+func MoveTodoDown(fileName string, sha string, action todo.TodoCommand) error {
+	todos, err := ReadRebaseTodoFile(fileName)
+	if err != nil {
+		return err
+	}
+
+	for i := range todos {
+		t := &todos[i]
+		// Comparing just the sha is not enough; we need to compare both the
+		// action and the sha, as the sha could appear multiple times (e.g. in a
+		// pick and later in a merge)
+		if t.Command == action && equalShas(t.Commit, sha) {
+			// The todos are ordered backwards compared to our model commits, so
+			// actually move the commit _up_ in the todos slice (i.e. towards 0)
```

Comment:
I wonder if it would simplify the code to go `todos = lo.Reverse(todos)` before doing anything else, and then reverse again at the end?